src/counter.rs

```diff
@@ -5,6 +5,7 @@ use std::sync::atomic::{AtomicUsize, Ordering};
 use timing;
 
 const DEFAULT_METRICS_RATE: usize = 100;
```

Comment:
Let's nuke DEFAULT_METRICS_RATE while we're here.  It's basically broken and we *always* set it to 1 in bash